examples/create.rs

```diff
@@ -46,7 +46,7 @@ fn main() {
 fn run() -> Result<()> {
     let args: Vec<String> = env::args().collect();
 
-    if args.len() != 3 {
+    if args.len() != 4 {
```

Comment:
Is this correct? Usage string below suggests args.len() == 3, and you only seem to access up to args[2]. It seems to me that either this check or usage below is correct, not both.

Reply:
Ouch, this is a leftover from checking the other auth methods. I will recommit now.